dub.json

```diff
@@ -6,5 +6,18 @@
 	"license": "MIT",
 	"dependencies": {
 	},
-	"excludedSourceFiles": ["source/dklib/benchmark_khash.d"]
+	"configurations": [
+		{
+			"name": "default",
+			"targetType": "library",
+			"excludedSourceFiles": ["source/dklib/benchmark_khash.d"]
+		},
+		{
+			"name": "bench",
+			"targetType": "executable",
```

Comment:
I like what you are trying to do here but note the header of the _benchmark file; it is an embedded dub.sdl and can be run directly from cmdline like a script.

It was my mistake to put it under `source/dklib`.  Please instead move it to a tests/ or examples/ (probably better) and document that it can be run from command line like script